packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -147,6 +166,17 @@ class InspectorTreeController extends Object
     setState(() {
       _root = node;
       _populateSearchableCachedRows();
+
+      ga.select(
```

Comment:
how often is this `set root` method called?

Reply:
- first time navigating to tab
- each time refresh is hit
- any time a widget is navigated to, the details tree may need to set it's root again. (if the selection is outside the current display.
- (I may have missed some others, but the above should catch the main ones :D)

Reply:
So it should essentially cover every time we are trying to show the user a new tree :)